tool/lsm_data.go

```diff
@@ -728,6 +727,11 @@ let version = {
                 humanize(data.Files[fileNum].Size) +
                 ")" +
                 overlapInfo +
+                "<" +
```

Comment:
could use a space before this <
just looking at the screenshot, I'm not sure how much benefit we're getting from enclosing it in < > but I don't have a better idea/strong opinions

Reply:
Done.